### PR TITLE
Default some required fields

### PR DESCRIPTION
**Pull request by grouma**
Closes https://github.com/dart-lang/webdev/issues/592
Closes https://github.com/dart-lang/webdev/issues/588
Closes https://github.com/dart-lang/webdev/issues/587

### CODE REVIEW

File: dwds/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.5.2
+
+- Fix issue where certain required fields of VM service protocol objects were
```
Comment (alan-knight) [inline dwds/CHANGELOG.md]: nit: line length

File: dwds/lib/src/debugging/instance.dart [hunk 2]
```diff
@@ -70,8 +70,7 @@ class InstanceHelper {
           ..decl = (FieldRef()
             // TODO(grouma) - Convert JS name to Dart.
             ..name = property.name
-            ..owner = classRef
-            ..declaredType = (InstanceRef()..classRef = ClassRef()))
+            ..owner = classRef)
```
Comment (alan-knight) [inline dwds/lib/src/debugging/instance.dart]: So the fieldRef doesn't need a declaredType?

Comment (grouma) [reply dwds/lib/src/debugging/instance.dart]: As far as I can tell no.

File: dwds/CHANGELOG.md (hunk 1 above)
Comment (grouma) [reply dwds/CHANGELOG.md]: I'm seeing this as under 80 for me?

File: dwds/lib/src/debugging/instance.dart (hunk 2 above)
Comment (alan-knight) [reply dwds/lib/src/debugging/instance.dart]: OK

**Pull request merged**